### PR TITLE
[wgsl-out] Write global initializers

### PR DESCRIPTION
**Pull request by JCapucho**

### CODE REVIEW

**Comment by JCapucho** [comment]
Hmm `spv-in` seems to be generating a wrong initializer

**Comment by JCapucho** [comment]
Oh no wait the problem is in `wgsl-out`, the structure has more fields

**Comment by JCapucho** [comment]
@Gordon-F Since you wrote the wgsl backend could you give this a review?

File: src/back/wgsl/writer.rs [hunk 1]
```diff
@@ -1377,8 +1384,21 @@ impl<W: Write> Writer<W> {
...
+                    _ => None,
+                };
+
                 // Write the comma separated constants
                 for (index, constant) in components.iter().enumerate() {
+                    if let Some(&Binding::BuiltIn(builtin)) =
+                        members.and_then(|members| members.get(index)?.binding.as_ref())
+                    {
+                        if builtin_str(builtin).is_none() {
+                            continue;
```
Comment (Gordon-F) [inline src/back/wgsl/writer.rs:1398]: @JCapucho Thank you! LGTM. Could you add `log::warn` here?

**Review by kvark — APPROVED** [review]
Thank you!

**Pull request merged**